editor/src/core/performance/performance-regression-tests.spec.tsx

```diff
@@ -64,7 +64,7 @@ describe('React Render Count Tests -', () => {
 
     const renderCountAfter = renderResult.getNumberOfRenders()
     // if this breaks, GREAT NEWS but update the test please :)
-    expect(renderCountAfter - renderCountBefore).toMatchInlineSnapshot(`402`)
+    expect(renderCountAfter - renderCountBefore).toMatchInlineSnapshot(`400`)
```

Comment:
Any idea why each of these dropped by 2? They seem very unrelated...

Reply:
Not particularly, I thought maybe it was returning nulls in something and now only returns 1, but glancing over it I can't particularly see anything like that.